git/client.go

```diff
@@ -19,6 +19,9 @@ import (
 	"github.com/cli/safeexec"
 )
 
+// MergeBaseConfig is the configuration setting to keep track of the PR target branch.
+const MergeBaseConfig = "gh-merge-base"
```

Comment:
**Question**

Is `merge` the best word for this name? Does the vscode extension use it for other things than the PR base? Can you think of a reason we would want to do something else in the future? If not, maybe `gh-pr-base` is clearer?

Reply:
It's about consistency not only with other tooling - both VSCode (the legacy name) and the GitHub Issues and Pull Requests extension - but also with the CLI's only wording: from `gh pr create --help`:

```text
  -B, --base branch          The branch into which you want your code merged
```

Ultimately, it's what branch your topic branch gets merged into. Not married to it, but I also see no strong reason to change it. I appreciate the cost of coordination may not be worth it, but I think sharing common wording helps understanding.